Flixel Changes

system32unknown opened the pull request:
- Remove `FlxRuntimeShader` since Haxeflixel added `FlxRuntimeShader` https://github.com/HaxeFlixel/flixel-addons/pull/399
- Remove `FlxAnimationController` and Implement `FlxG.animationTimeScale`
- Remove `StdLibs.cpp` since hxcpp fixed hex colors
* Fix `FlxCamera.filters` Deprecation Warning https://github.com/HaxeFlixel/flixel/pull/2890

system32unknown committed:
flixel changes

- Remove `FlxRuntimeShader` since Haxeflixel added `FlxRuntimeShader` https://github.com/HaxeFlixel/flixel-addons/pull/399
* Migrate `FlxAnimationController` to Flixel 5.4.1 https://github.com/HaxeFlixel/flixel/pull/2930
* Fix `FlxCamera.filters` Deprecation Warning https://github.com/HaxeFlixel/flixel/pull/2890

CaptainBaldi commented:
Looks pretty good, however I would recommend maybe just making a Pull Request on Flixel that makes `FlxAnimationController.timeScale` static, that way if it was merged, we could make another PR that removes the use of a custom `FlxAnimationController` entirely.

CaptainBaldi commented:
> Looks pretty good, however I would recommend maybe just making a Pull Request on Flixel that makes `FlxAnimationController.timeScale` static, that way if it was merged, we could make another PR that removes the use of a custom `FlxAnimationController` entirely.



> Looks pretty good, however I would recommend maybe just making a Pull Request on Flixel that makes `FlxAnimationController.timeScale` static, that way if it was merged (and whenever the next Flixel update would release), we could make another PR that removes the use of a custom `FlxAnimationController` entirely.

system32unknown committed:
Flixel Changes 2

 - Remove `FlxAnimationController` and Implement `FlxG.animationTimeScale`
- Remove 'StdLibs.cpp' since hxcpp fixed hex colors
